src/coreclr/src/System.Private.CoreLib/src/System/String.CoreCLR.cs

```diff
@@ -26,6 +26,7 @@ public partial class String
         [System.Runtime.CompilerServices.IndexerName("Chars")]
         public extern char this[int index]
         {
+            [System.Runtime.CompilerServices.Intrinsic]
```

Comment:
Nit: There is `using System.Runtime.CompilerServices` at the top of the file. This can be just `[Intrinsic]`

Reply:
Fixed in 6th commit.